grails-app/services/au/org/ala/ecodata/SiteService.groovy

```diff
@@ -255,6 +255,29 @@ class SiteService {
         site
     }
 
+    def getSimpleProjectArea(projectSiteId) {
+        def threshold = grailsApplication.config.getProperty('biocollect.projectArea.simplificationThreshold', 10000)
+        def tolerance = grailsApplication.config.getProperty('biocollect.projectArea.simplificationTolerance', 0.1)
```

Comment:
As per above, 0.1 seems a bit high to use as a default.  I'd probably default to ~10m - you can normally get pretty good size reductions from that.  If you are feeling keen, could start at 10m then while number of points > 10000, multiply tolerance by 10, try again.

Reply:
Updated the tolerance to be ~10m by default 👍🏻